toolkit/docs/formats/imageconfig.md

```diff
@@ -337,6 +337,11 @@ The Security Enhanced Linux (SELinux) feature is enabled by using the `SELinux`
 This will instruct init (systemd) to set the configured mode on boot.  The `force_enforcing` option will set enforcing in the config and also add `enforcing=1` in the kernel command line,
 which is a higher precedent than the config file. This ensures SELinux boots in enforcing even if the /etc/selinux/config was altered.
 
+The version for CGroup in Mariner images can be enabled by using the `CGroup` key with value containing which version to use on boot. The value that can be chosen is either `version_one` or `version_two`. 
+The `version_two` value will set the cgroupv2 to be used in Mariner by setting the config value `systemd.unified_cgroup_hierarchy=1` in /boot/systemd.cfg. The value `version_one` or no value set will keep
```

Comment:
```suggestion
The `version_two` value will set the cgroupv2 to be used in Mariner by setting the config value `systemd.unified_cgroup_hierarchy=1` in the default kernel command line. The value `version_one` or no value set will keep
```
nit - not sure if the exact detail for /boot/systemd.cfg should be explained. Ultimately, the change adds `systemd.unified_cgroup_hierarchy=1` into the kernel command line, which today is implemented by grub.cfg's import of /boot/systemd.cfg to set an env var, which is used in the actual kernel command line arguments. But the overall expected behavior of this is to have `systemd.unified_cgroup_hierarchy=1` present in the kernel cmdline when `version_two` value is used.